homeassistant/components/syncthru/__init__.py

```diff
@@ -91,6 +91,6 @@ def device_connections(printer: SyncThru) -> set[tuple[str, str]]:
         mac = printer.raw()["identity"]["mac_addr"]
```

Comment:
If you know that the reason its going to throw is that `mac_addr` is missing, check that it exists instead of wrapping it in a try.

Reply:
good point